packages/lodestar/src/chain/utils.ts

```diff
@@ -0,0 +1,9 @@
+import {BeaconBlock} from "@chainsafe/lodestar-types";
+
+export function getBlockAttestationCount(block: BeaconBlock): number {
+  return Array.from(
```

Comment:
If this is to be used, and not just commented/used in debug settings, we should use readOnlyMap here instead of Array.from.

Reply:
That just for debugging. Too many iterations to keep it in log method